src/components/Editor/EditorTabs.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Tab } from "@/types";
+import "./EditorTabs.css";
+import EditorTab from "./EditorTab";
+
+interface EditorTabsProps {
+  selectedTab: Tab;
+  onTabChange?: (newTab: Tab) => void;
+}
+
+const EditorTabs: React.FC<EditorTabsProps> = ({
+  selectedTab,
+  onTabChange,
+}) => (
+  <div
+    className="tabs-panel"
+    style={{
+      borderRight: "1px solid rgba(255, 255, 255, 0.1)",
+      width: 50,
+    }}
+  >
+    <EditorTab
+      title="JSX"
+      selected={selectedTab === "jsx"}
+      onClick={() => onTabChange("jsx")}
```

Comment:
I'm wondering if we can expand this to any language/extension, so JSX can be just "code"